Makefile

```diff
@@ -31,7 +31,7 @@ compress:
 	upx --brute -1 ./dist/carrier-darwin-amd64
 
 test: lint
-	ginkgo ./cmd/internal/client/ ./tools/ ./helpers/ ./kubernetes/
+	ginkgo ./cli/internal/client/ ./tools/ ./helpers/ ./kubernetes/
```

Comment:
This looks to be out of date. The full set of directories inside of which I found `*_test.go` (*) files is

  - acceptance
  - helpers
  - internal/cli
  - internal/services
  - kubernetes

(*) Command used: `find . -type f -name '*_test.go'|xargs -n1 dirname | uniq | sort`

Reply:
Done with commit 73aa2f0b7413d271dc64a99759e9ea1d9f3909c2